src/app/client/src/app/modules/workspace/components/workspace-content-filter/workspace-content-filter.component.ts

```diff
@@ -125,13 +126,28 @@ export class AllMyContentFilterComponent implements OnInit {
         pageid: 'all-my-content-page'
       };
   }
+
+  setFilterEnv() {
```

Comment:
the method name is not correct. it should be named as **setFilterTypeAndRedirectURL**